Source/Fuse.Common/Timer.uno

```diff
@@ -22,15 +22,21 @@ namespace Fuse
 
 		void Start()
 		{
-			_startTime = Uno.Diagnostics.Clock.GetSeconds();
-			UpdateManager.AddAction(Update);
-			_running = true;
+			if(!_running)
+			{
+				_startTime = Uno.Diagnostics.Clock.GetSeconds();
+				UpdateManager.AddAction(Update);
+				_running = true;
+			}
```

Comment:
Some kind of state machine is needed in order to stop it from calling Stop() twice(f.ex once by the callback calling dispose, once by itself calling stop). 
My suggested fix is to change Update() to call Dispose, and having a check in Dispose, returning if the timer isn't running